src/experiments.js

```diff
@@ -37,6 +42,9 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 /** @type {Object<string, boolean>} */
 let toggles_ = null;
 
+/** @type {!Promise} */
+const originTrialsPromise = enableExperimentsForOriginTrials(self);
```

Comment:
We should avoid competing with startup tasks. The non-startup flavor of `chunk` may be a good fit here. 

How long does it take to verify a single token? That should decide the granularity of the chunk.